[BUG] [ESP32] Kconfig error when setting esp32h2 target

### Reproduction steps

```bash
1. Checkout v5.0-beta1 tag for esp-idf
2. At lighting-app example path in matter repo, setting esp32h2 target will get an error.

idf.py --preview set-target esp32h2 
...
  greatest_range = [int(min_range.str_value, base), int(max_range.str_value, base)]
ValueError: invalid literal for int() with base 10: ''
CMake Error at /home/wangqixiang/workspace/gitlab-esp/esp-idf/tools/cmake/kconfig.cmake:176 (message):
```
```


### Bug prevalence

Whenever I do this

### GitHub hash of the SDK that was being used

3aa138f3425f2b0e611a05d68c3275b791ef499e

### Platform

esp32

### Platform Version(s)

_No response_

### Anything else?

_No response_

--- a/config/esp32/components/chip/Kconfig
+++ b/config/esp32/components/chip/Kconfig
@@ -520,25 +520,30 @@ menu "CHIP Device Layer"
         visible if ENABLE_ETHERNET_TELEMETRY
 
         config GPIO_RANGE_MIN
-            int 
+            int
+            depends on ENABLE_ETHERNET_TELEMETRY
             default 0
 
         config GPIO_RANGE_MAX
-            int 
+            int
+            depends on ENABLE_ETHERNET_TELEMETRY
             default 33 if IDF_TARGET_ESP32
             default 46 if IDF_TARGET_ESP32S2
             default 19 if IDF_TARGET_ESP32C3
             default 48 if IDF_TARGET_ESP32S3
+            default 26 if IDF_TARGET_ESP32H2
 
         config ETH_MDC_GPIO
             int "SMI MDC GPIO number"
+            depends on ENABLE_ETHERNET_TELEMETRY
             range GPIO_RANGE_MIN GPIO_RANGE_MAX
-            default 23 if ENABLE_ETHERNET_TELEMETRY 
+            default 23 if ENABLE_ETHERNET_TELEMETRY
             help
                 Set the GPIO number used by SMI MDC.
 
         config ETH_MDIO_GPIO
             int "SMI MDIO GPIO number"
+            depends on ENABLE_ETHERNET_TELEMETRY
             range GPIO_RANGE_MIN GPIO_RANGE_MAX
             default 18 if ENABLE_ETHERNET_TELEMETRY
             help
@@ -546,6 +551,7 @@ menu "CHIP Device Layer"
 
         config ETH_PHY_RST_GPIO
             int "PHY Reset GPIO number"
+            depends on ENABLE_ETHERNET_TELEMETRY
             range -1 GPIO_RANGE_MAX
             default 5 if ENABLE_ETHERNET_TELEMETRY
             help

